app/src/controllers/log/sagas.js

```diff
@@ -104,13 +105,14 @@ function* fetchStackTrace({ payload: logItem }) {
   yield take(createFetchPredicate(STACK_TRACE_NAMESPACE));
 }
 
-function* fetchHistoryItems() {
+function* fetchHistoryItems(action) {
+  const payload = (action && action.payload) || HISTORY_LINE_DEFAULT_VALUE;
```

Comment:
It can be specified in the function parameters as default value, f.e.:
`fetchHistoryItems(action = { payload: HISTORY_LINE_DEFAULT_VALUE  })`